src/main/kotlin/com/vaadin/gradle/VaadinBuildFrontendTask.kt

```diff
@@ -58,6 +58,14 @@ open class VaadinBuildFrontendTask : DefaultTask() {
         project.tasks.withType(Jar::class.java) { task: Jar ->
             task.mustRunAfter("vaadinBuildFrontend")
         }
+
+        setOnlyIf {
+            val productionMode: Boolean = VaadinFlowPluginExtension.get(project).productionMode
+            if (!productionMode) {
+                logger.info("Skipping vaadinBuildFrontend since we're not in production mode")
+            }
+            productionMode
```

Comment:
As noted in documentation this should preferably also be enabled if `enableDevServer` is false as that would be the same as executing both goals in maven without productionMode=true.

Reply:
Ah, the `enableDevServer` from the `flow-build-info.json` file. Understood, should now be fixed.